javascript-typing-tutor/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <p><span class='none underline'>g</span>
+      <span class='none'>r</span>
+      <span class='none'>u</span>
+      <span class='none'>m</span>
+      <span class='none'>p</span>
+      <span class='none'>y</span>
+      <span class='none'> &nbsp;</span>
+      <span class='none'>w</span>
+      <span class='none'>i</span>
+      <span class='none'>z</span>
+      <span class='none'>a</span>
+      <span class='none'>r</span>
+      <span class='none'>d</span>
+      <span class='none'>s</span>
+      <span class='none'> &nbsp; </span>
+      <span class='none'>m</span>
+      <span class='none'>a</span>
+      <span class='none'>k</span>
+      <span class='none'>e</span>
+      <span class='none'>&nbsp;</span>
+      <span class='none'>t</span>
+      <span class='none'>o</span>
+      <span class='none'>x</span>
+      <span class='none'>i</span>
+      <span class='none'>c</span>
+      <span class='none'>&nbsp;</span>
+      <span class='none'>b</span>
+      <span class='none'>r</span>
+      <span class='none'>e</span>
+      <span class='none'>w</span></p>
+    </div>
```

Comment:
For some reason, my note didn't go through last time 🤔 . My note was, what happens if you use regular space characters instead of `&nbsp;`?

Reply:
![image](https://user-images.githubusercontent.com/11698908/124962842-f36c8100-dfd3-11eb-9643-285617d710e2.png)
![image](https://user-images.githubusercontent.com/11698908/124962896-02533380-dfd4-11eb-9356-f2ca54a42e10.png)
I'm not sure what you mean by regular space characters. But white space is collapsed by HTML. 
In the screen shots the span between s and m is collapsed by the browser.